frontend/package.json

```diff
@@ -62,6 +62,7 @@
     "@iconify-json/oi": "^1.1.8",
     "@pinia/testing": "0.1.3",
     "@popperjs/core": "^2.11.8",
+    "@rollup/rollup-darwin-arm64": "^4.18.0",
```

Comment:
storybook didn't start without me adding this. Will look at it again, since this is now breaking CI